bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableClient.scala

```diff
@@ -131,7 +131,6 @@ trait CachedBackend extends VoTableBackend {
 
   // Cache the query not the future so that failed queries are executed again
   override protected [votable] def doQuery(query: CatalogQuery, url: URL)(ec: ExecutionContext): Future[QueryResult] = Future {
-    Logger.getLogger(getClass.getName).info(s"Starting BAGS lookup on ${Thread.currentThread}")
```

Comment:
I'm not sure either, we could lower it to fine. My main complaint is the BAGS mention as this is not a BAGS-only event